installer/ansible/script/sdsrc

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Global
+HOST_IP=${HOST_IP:-}
+HOST_IP=$(get_default_host_ip "$HOST_IP" "inet")
+if [ "$HOST_IP" == "" ]; then
+    die $LINENO "Could not determine host ip address. See local.conf for suggestions on setting HOST_IP."
+fi
+
+# OpenSDS configuration.
+OPENSDS_VERSION=${OPENSDS_VERSION:-v1beta}
+
+# Multi-cloud 
+MULTICLOUD_VERSION=${MULTICLOUD_VERSION:-v1}
+
+# OpenSDS service name in keystone.
+OPENSDS_SERVER_NAME=${OPENSDS_SERVER_NAME:-opensds}
+
+# Multi-Cloud service name in keystone
+MULTICLOUD_SERVER_NAME=${MULTICLOUD_SERVER_NAME:-opensds}
+
+# devstack keystone configuration
```

Comment:
line 36-45 not required. Please check

Reply:
also line # 24-25, 30-31.

Reply:
removed unused variables. most of them are being used in keystone.sh

https://github.com/sodafoundation/multi-cloud/blob/b6459548812ed643e9a0e8f1186f5d341e86cab2/installer/ansible/script/keystone.sh